packages/components/src/FormatFile/FormatFile.tsx

```diff
@@ -48,7 +48,7 @@ export function FormatFile({
 }: FormatFileProps) {
   const isComplete = file.progress >= 1;
   const fileSize = getHumanReadableFileSize(file.size);
-  const wrapperClassNames = classnames(styles[display], {
+  const wrapperClassNames = classnames(styles[display], styles.formatFile, {
```

Comment:
created this new class as the others being applied at this layer are specific to certain variations of the component (size/presentation)